test/scripts/e2e-cp.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -ex
+
+pushd e2e
+go mod download
+popd
+
+export KUBERNETES_CONFORMANCE_TEST=y
+export KUBECONFIG=${HOME}/admin.conf
+export MASTER_NAME=${KIND_CLUSTER_NAME}-control-plane
+export NODE_NAMES=${MASTER_NAME}
+export KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS}
+
+sed -E -i 's/"\$\{ginkgo\}" "\$\{ginkgo_args\[\@\]\:\+\$\{ginkgo_args\[\@\]\}\}" "\$\{e2e_test\}"/pushd \$GITHUB_WORKSPACE\/test\/e2e\nGO111MODULE=on "\$\{ginkgo\}" "\$\{ginkgo_args\[\@\]\:\+\$\{ginkgo_args\[\@\]\}\}"/' ${GOPATH}/src/k8s.io/kubernetes/hack/ginkgo-e2e.sh
```

Comment:
@dave-tucker what is this?

Reply:
I see you just copied this over and it was already here:
https://github.com/ovn-org/ovn-kubernetes/pull/1207/files#diff-4ab65103bef547df91af90c686cea90aR15

@as-com what is this doing?

I wont hold up this PR because of it, but we should probably try to make this more simple if we can...or at least document what this line does.

Reply:
This nasty regex patches the ginkgo-e2e.sh file from Kubernetes to run _our_ e2e test suite instead of Kubernetes's. This shell script is hardcoded into kubetest, which does some amount of magic on its own, so I figured that this would be one of the better ways to run our tests.

Reply:
@as-com as we don't really change kubetest. We could just fork it and make a modification in the code, build it and store it in artifacts. Would that work for our e2e suite?